pyanaconda/modules/network/nm_client.py

```diff
@@ -788,6 +792,23 @@ def update_connection_ip_settings_from_ksdata(connection, network_data):
                 log.error("IP address %s is not valid", ns)
 
 
+def update_connection_wired_settings_from_ksdata(connection, network_data):
+    """Update NM connection wired settings from kickstart in place.
+
+    :param connection: existing NetworkManager connection to be updated
+    :type connection: NM.RemoteConnection
+    :param network_data: kickstart configuation to be applied to the connection
+    :type network_data: pykickstart NetworkData
+    """
+    # mtu can be configured also on virtual device connections
+    if network_data.mtu:
+        s_wired = connection.get_setting_wired()
+        if not s_wired:
+            s_wired = NM.SettingWired.new()
+            connection.add_setting(s_wired)
+        s_wired.props.mtu = int(network_data.mtu)
```

Comment:
Thank you,
I hoped that such value would not make it so far and neglected this possibility, but actually it would.
Unit tests would make me think about it more:)
I'll update the PR and add unit test for the new function.

Reply:
Ok. The value should be ideally validated and converted by pykickstart if that's possible.

Reply:
Unit test added, int conversion checked.